index.html

```diff
@@ -4,42 +4,57 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <script src="script.js"></script>
+    <title>Calculator</title>
+    <script src="script.js" defer></script>
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <div class="mainWrapper">
       <div class="windowWrapper">
-        <div class="inputWrapper">
-          <input class="inputWindow" type="text" />
+        <div class="topWrapper">
+          <div class="inputNumber">0</div>
         </div>
-        <!-- <div class="operatorWrapper">
-          <div class="operatorTop">
-            <button class="placeholder">+</button>
-            <button class="placeholder">-</button>
-            <button class="placeholder">*</button>
-            <button class="placeholder">/</button>
-          </div>
-        </div> -->
+
         <div class="numpadWrapper">
           <div class="topNumpad">
-            <button class="placeholder">7</button>
-            <button class="placeholder">8</button>
-            <button class="placeholder">9</button>
-            <button class="placeholder">4</button>
-            <button class="placeholder">5</button>
-            <button class="placeholder">6</button>
-            <button class="placeholder">1</button>
-            <button class="placeholder">2</button>
-            <button class="placeholder">3</button>
+            <button class="calcButton" id="button-side" data-clear>CA</button>
+            <button class="calcButton" id="button-side" data-plusminus>
+              +/-
+            </button>
+            <button class="calcButton" id="button-side" data-delete>⌫</button>
+            <button class="calcButton division" id="button-side" data-operation='division'>
+              ÷
+            </button>
           </div>
           <div class="bottomNumpad">
-            <button class="placeholder">0</button>
-            <button class="placeholder">,</button>
+            <div class="numbers">
+              <button class="calcButton" data-number='9'>9</button>
+              <button class="calcButton" data-number='8'>8</button>
+              <button class="calcButton" data-number='7'>7</button>
+              <button class="calcButton" data-number='6'>6</button>
+              <button class="calcButton" data-number='5'>5</button>
+              <button class="calcButton" data-number='4'>4</button>
+              <button class="calcButton" data-number='3'>3</button>
+              <button class="calcButton" data-number='2'>2</button>
+              <button class="calcButton" data-number='1'>1</button>
+              <button class="calcButton" data-number='.'>.</button>
+              <button class="calcButton" id="button-wide" data-number='0'>0</button>
+            </div>
+            <div class="operators">
+              <button class="calcButton multiplication" id="button-side" data-operation='multiplication'>
```

Comment:
```suggestion
              <button class="calcButton button-side" id="multiplication" data-operation='multiplication'>
```

I think the usage of classes vs id is suboptimal here. IMO `multiplication` should rather be an ID whereas `button-side` should be a class. It's occuring multiple times anyway, which is not allowed for IDs.

Reply:
+1, but I would just stick to classes only.

Reply:
I've implemented Marc's suggestion for now.